backend-project/small_eod/tags/tests.py

```diff
@@ -2,25 +2,22 @@
 
 from .factories import TagFactory, TagNamespaceFactory
 from .models import Tag, TagNamespace
-from ..generic.tests import FactoryTestCase
+from ..generic.tests import FactoryCreateObjectsMixin
 
 
-class TagFactoryTestCase(FactoryTestCase):
+class TagFactoryTestCase(FactoryCreateObjectsMixin, TestCase):
+    FACTORY = (Tag, TagFactory)
```

Comment:
To już drobiazg naprawdę, ale dlaczego nie rozbić tego na dwa pola tj. ```model```, ```factory_class``` podobnie jak w https://github.com/watchdogpolska/small_eod/blob/dev/backend-project/small_eod/generic/tests.py#L6-L8 ?